.devcontainer/readme.md

```diff
@@ -21,11 +21,11 @@ Install/update VS Code extension "Remote - Containers".
 
 ## How to use the container
 
-Open your AppUI repo in VS Code, click the green area in the status bar, and select VS Code command 'Remote-Containers: Reopen Folder in Container'. Once connected, the green section of the status bar should say something like "Dev Container: appui-linux".
```

Comment:
Do we really want to go with `AppUi` instead of `AppUI`?

Reply:
I initially got with the `UI`, but the documentation refers to it as `Ui`, I'm not sure what the rules are in english.

Reply:
I'd vote for `AppUI`, since it's an acronym? Also that's consistent w/ https://github.com/iTwin/iTwinUI

+@NancyMcCallB

Reply:
I'll update accordingly.

Reply:
Updated